[2022-03-18 19:49:55,468] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2022-03-18T14:18:08.915997+00:00 [queued]>
[2022-03-18 19:49:55,520] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2022-03-18T14:18:08.915997+00:00 [queued]>
[2022-03-18 19:49:55,520] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 19:49:55,521] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-03-18 19:49:55,522] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 19:49:55,539] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2022-03-18T14:18:08.915997+00:00
[2022-03-18 19:49:55,546] {standard_task_runner.py:52} INFO - Started process 28842 to run task
[2022-03-18 19:49:55,553] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2022-03-18T14:18:08.915997+00:00', '--job-id', '28', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpfs7sizbs', '--error-file', '/tmp/tmp6gg76lwn']
[2022-03-18 19:49:55,557] {standard_task_runner.py:77} INFO - Job 28: Subtask task_4
[2022-03-18 19:49:55,642] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2022-03-18T14:18:08.915997+00:00 [running]> on host pc
[2022-03-18 19:49:55,765] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2022-03-18T14:18:08.915997+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-18T14:18:08.915997+00:00
[2022-03-18 19:49:55,767] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-03-18 19:49:55,767] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-03-18 19:49:55,778] {subprocess.py:75} INFO - Output:
[2022-03-18 19:49:58,782] {subprocess.py:83} INFO - Command exited with return code 0
[2022-03-18 19:49:58,935] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20220318T141808, start_date=20220318T141955, end_date=20220318T141958
[2022-03-18 19:49:59,104] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-18 19:49:59,147] {local_task_job.py:151} INFO - Task exited with return code 0
